xpdacq/xpdacq.py

```diff
@@ -179,7 +179,7 @@ def _auto_dark_collection(scan):
     except KeyError:
         # protection, shouldn't happen
         warnings.warn("It seems your ScanPlan object wasn't instantiated properly."
-                        "This scan will keep going but please check your package after this scan")
+                        "This scan will keep going but please contact instrument scientist immediately")
```

Comment:
This message is still not quite right.  Please make it this:

`` "It seems your ScanPlan object wasn't instantiated properly. This may indicate a problem with the current version of the code."
 -                        "This scan will keep going but please notify the instrument scientist who can post a bug report"